packages/helm-charts/testnet/templates/validators.statefulset.yaml

```diff
@@ -136,22 +136,22 @@ spec:
       containers:
       - name: geth
         image: {{ .Values.geth.image.repository }}:{{ .Values.geth.image.tag }}
-        imagePullPolicy: IfNotPresent
+        imagePullPolicy: Always
         command: ["/bin/sh"]
         args:
         - "-c"
         - |-
           set -euo pipefail
-          # Delete the private key as it is no longer needed
-          rm /root/.celo/pkey
+          # Don't delete the private key as it is still needed
```

Comment:
What was the issue here? I don't see where it's used

Reply:
Ah, you're right. It isn't used.  I was seeing Validator0 failing to start with `rm: can't remove '/root/.celo/pkey': No such file or directory`.  But now I see that was because of another error.  When it was automatically restarted, the same code ran twice, and it produced that error (obscuring the root cause). I updated so it still leaves pkey, but revised explanation in comment.